models/User.php

```diff
@@ -990,6 +990,15 @@ public function getKeyBindings()
         return $this->keyBindings ? $this->keyBindings : self::getDefaultKeyBindings();
     }
 
+    public static function strictKeybinds(array $bindings): array
```

Comment:
Improved types (for @blankse - see #13996):

```suggestion
    /**
     * @param list<array{action: string, key: int, alt?: bool, ctrl?: bool, shift?: bool}> $bindings
     *
     * @return list<array{action: string, key: int, alt: bool, ctrl: bool, shift: bool}>
     */
    public static function strictKeybinds(array $bindings): array
```